iceoryx_posh/include/iceoryx_posh/internal/mepoo/segment_manager.hpp

```diff
@@ -49,7 +50,9 @@ class SegmentManager
     struct SegmentMapping
     {
       public:
-        SegmentMapping(const std::string& sharedMemoryName,
+        using string_t = cxx::string<128>;
```

Comment:
No constexpr for `128`?

Reply:
Magic Number? I think the shared memory needs also a `MAX_SHARED_MEMORY_NAME_LENGTH` variable in `posh_types`

Reply:
I don't consider this as magic number. It's obvious what the number means in the context of the string and we also don't have it for most of the string aliases. The constexpr was only used when the value of the capacity was needed. I'd rather create a static `capacity()` function in the `cxx::string` than to have everywhere a constexpr for the length. Just in case, you also don't want a constexpr for the length of every vector, do you?

Reply:
Actually, I would disagree. And here the argument is not that the user does not know what the `128` means it is more like we would like to change sometime the support name length and then we find every property in the posh_types file except this one which  is stored in some header file.
I think now that we have started the thing with the posh_types we should be consistent.

Reply:
I agree with @elfenpiff . Maybe we don't need a `constexpr` for every single one and use a generic/default one like `MAX_DEFAULT_NAME_LENGTH` for all `128` for example?

Reply:
@elfenpiff `ProcessName_t` **is** stored in the posh_types ;)

Reply:
@elfenpiff in this case, the alias should probably live there. There is an item in #260 which takes care of the convention on how to name string aliases and I wanted to do it when we decide on that one.